Cargo.toml

```diff
@@ -19,5 +19,9 @@ serde = "0.8"
 uuid = {version = "0.3", features = ["v4"]}
 fnv = "1.0.3"
 
+[target.'cfg(target_os = "linux")'.dependencies]
+mio = "0.6.1"
+fnv = "1.0.3"
```

Comment:
I think it is only used in the `unix` module so perhaps it should be moved here?